nautobot/cloud/forms.py

```diff
@@ -72,6 +75,95 @@ class CloudAccountFilterForm(NautobotFilterForm):
     tags = TagFilterField(model)
 
 
+#
+# Cloud Network
+#
+
+
+class CloudNetworkForm(NautobotModelForm):
+    cloud_type = DynamicModelChoiceField(
+        queryset=CloudType.objects.all(),
+    )
+    cloud_account = DynamicModelChoiceField(
+        queryset=CloudAccount.objects.all(),
+    )
+    parent = DynamicModelChoiceField(
+        queryset=CloudNetwork.objects.all(),
+        required=False,
+    )
+    namespace = DynamicModelChoiceField(queryset=Namespace.objects.all(), required=False)
+    prefixes = DynamicModelMultipleChoiceField(
+        queryset=Prefix.objects.all(),
+        required=False,
+        query_params={"namespace": "$namespace"},
+    )
+
+    class Meta:
+        model = CloudNetwork
+        fields = [
+            "name",
+            "description",
+            "cloud_type",
+            "cloud_account",
+            "parent",
+            "prefixes",
+            "extra_config",
+            "tags",
+        ]
+        EXTRA_CONFIG_HELP_TEXT = """
+            Optional user-defined <a href="https://json.org/">JSON</a> data for this integration. Example:
```

Comment:
Does `extra_config` field need to be marked as `required=False` or is that automatic?

Reply:
It defaults to `null`